CHANGELOG.md

```diff
@@ -0,0 +1,499 @@
+  * macOS: Try to find libPCBUSB.dylib before loading it (#983, #984)
+  * Disable command PCAN_ALLOW_ERROR_FRAMES on macOS (#985)
+  * Force english error messages (#986, #993, #994)
+  * Add set/get device number (#987)
+  * Timestamps are silently incorrect on Windows without uptime installed (#1053, #1093)
+  * Implement check for minimum version of pcan library (#1065, #1188)
+  * Handle case where uptime is imported successfully but returns None (#1102, #1103)
+* slcan
+  * Fix bitrate setting (#691)
+  * Fix fileno crash on Windows (#924)
+* ics_neovi
+  * Filter out Tx error messages (#854)
+  * Adding support for send timeout (#855)
+  * Raising more precise API error when set bitrate fails (#865)
+  * Avoid flooding the logger with many errors when they are the same (#1125)
+  * Omit the transmit exception cause for brevity (#1086)
+  * Raise ValueError if message data is over max frame length (#1177, #1181)
+  * Setting is_error_frame message property (#1189)
+* ixxat
+  * Raise exception on busoff in recv() (#856)
+  * Add support for 666 kbit/s bitrate (#911)
+  * Add function to list hwids of available devices (#926)
+  * Add CAN FD support (#1119)
+* seeed
+  * Fix fileno crash on Windows (#902)
+* kvaser
+  * Improve timestamp accuracy on Windows (#934, #936)
+* usb2can
+  * Fix "Error 8" on Windows and provide better error messages (#989)
+* serial
+  * Fix "TypeError: cannot unpack non-iterable NoneType" and more robust error handling (#1000, #1010)
+* canalystii
+  * Fix is_extended_id (#1006)
+  * Fix transmitting onto a busy bus (#1114)
+  * Replace binary library with python driver (#726, #1127)
+
+Other API changes and improvements
+----------------------------------
+
+* CAN FD frame support is pretty complete (#963)
+  * ASCWriter (#604) and ASCReader (#741)
+  * Canutils reader and writer (#1042)
+  * Logger, viewer and player tools can handle CAN FD (#632)
+  * Many bugfixes and more testing coverage
+* IO
+  * Log rotation (#648, #874, #881, #1147)
+  * Add [plugin support to can.io Reader/Writer](https://python-can.readthedocs.io/en/develop/listeners.html#listener) (#783)
+  * ASCReader/Writer enhancements (#820)
+  * Adding absolute timestamps to ASC reader (#761)
+  * Support other base number (radix) at ASCReader (#764)
+  * Add [logconvert script](https://python-can.readthedocs.io/en/develop/scripts.html#can-logconvert) (#1072, #1194)
+  * Adding support for gzipped ASC logging file (.asc.gz) (#1138)
+  * Improve [IO class hierarchy](https://python-can.readthedocs.io/en/develop/internal-api.html#module-can.io.generic) (#1147)
+* An [overview over various "virtual" interfaces](https://python-can.readthedocs.io/en/develop/interfaces/virtual.html#other-virtual-interfaces) (#644)
+* Make ThreadBasedCyclicSendTask event based & improve timing accuracy (#656)
+* Ignore error frames in can.player by default, add --error-frames option (#690)
+* Add __eq__ method to can.Message (#737, #747)
```

Comment:
`Message.__eq__` is gone. No need to mention it here.

Reply:
```suggestion

```